proposals/lip-0053.md

```diff
@@ -519,7 +577,7 @@ def verify(ccu: Transaction) -> None:
         raise Exception("The first CCU must contain a non-empty certificate.")
     if len(certificateBytes) > 0:
         verifyCertificate(ccu)
-    if len(ccu.params.activeValidatorsUpdate) > 0 or ccu.params.certificateThreshold != validators(ccu.params.sendingChainID).certificateThreshold:
+    if len(ccu.params.blsKeysUpdate) > 0 or len(ccu.params.bftWeightsUpdate) > 0 or ccu.params.bftWeightsUpdateBitmap != EMPTY_BYTES or ccu.params.certificateThreshold != validators(ccu.params.sendingChainID).certificateThreshold:
```

Comment:
To be in sync with the updated CCU schema, we need to use `ccu.params.activeValidatorsUpdate.blsKeysUpdate` and  `ccu.params.activeValidatorsUpdate.bftWeightsUpdate` instead of  `ccu.params.blsKeysUpdate` and `ccu.params.bftWeightsUpdate`. (there are 3 places in the LIP where those variables appear).